book/website/_bibliography/references.bib

```diff
@@ -2983,3 +2983,118 @@ @article{michlmayr2009community
+  ISSN={1467-8535},
+  url={http://dx.doi.org/10.1111/bjet.13246},
+  DOI={10.1111/bjet.13246},
+  number={5},
+  journal={British Journal of Educational Technology},
+  publisher={Wiley},
+  author={Gebre,  Engida},
+  year={2022},
+  month={jun},
+  pages={1080–1095}
+}
+
+@article{Jiang2022datascience,
+  title={Data science education across the disciplines: Underexamined opportunities for K‐12 innovation},
+  volume={53},
+  ISSN={1467-8535},
+  url={http://dx.doi.org/10.1111/bjet.13258},
+  DOI={10.1111/bjet.13258},
+  number={5},
+  journal={British Journal of Educational Technology},
+  publisher={Wiley},
+  author={Jiang,  Shiyan and Lee,  Victor R. and Rosenberg,  Joshua M.},
+  year={2022},
+  month={jul},
+  pages={1073–1079}
+}
+
+@inproceedings{Kandlhofer2016artificial,
+  title={Artificial intelligence and computer science in education: From kindergarten to university},
+  url={http://dx.doi.org/10.1109/fie.2016.7757570},
+  DOI={10.1109/fie.2016.7757570},
+  booktitle={2016 IEEE Frontiers in Education Conference (FIE)},
+  publisher={IEEE},
+  author={Kandlhofer,  Martin and Steinbauer,  Gerald and Hirschmugl-Gaisch,  Sabine and Huber,  Petra},
+  year={2016},
+  month={oct},
+  pages={1–9}
+}
+
+@article{McFarland2021education,
+  title={Education Data Science: Past,  Present,  Future},
+  volume={7},
+  ISSN={2332-8584},
+  url={http://dx.doi.org/10.1177/23328584211052055},
+  DOI={10.1177/23328584211052055},
+  journal={AERA Open},
+  publisher={SAGE Publications},
+  author={McFarland,  Daniel A. and Khanna,  Saurabh and Domingue,  Benjamin W. and Pardos,  Zachary A.},
+  year={2021},
+  month={jan} 
+}
+
+@article{Stanja2022formative,
+  title={Formative assessment strategies for students’ conceptions—The potential of learning analytics},
+  volume={54},
+  ISSN={1467-8535},
+  url={http://dx.doi.org/10.1111/bjet.13288},
+  DOI={10.1111/bjet.13288},
+  number={1},
+  journal={British Journal of Educational Technology},
+  publisher={Wiley},
+  author={Stanja,  Judith and Gritz,  Wolfgang and Krugel,  Johannes and Hoppe,  Anett and Dannemann,  Sarah},
+  year={2022},
+  month={nov},
+  pages={58–75}
+}
+
+@book{tomlinson2017differentiate,
+  title={How to Differentiate Instruction in Academically Diverse Classrooms, Third Edition},
+  author={Tomlinson, C.A.},
+  isbn={9781416623304},
+  lccn={2016049811},
+  url={https://books.google.co.uk/books?id=zoh2DgAAQBAJ},
+  year={2017},
+  publisher={ASCD}
+}
+
+@article{Zhai2020meta,
+  title={A Meta-Analysis of Machine Learning-Based Science Assessments: Factors Impacting Machine-Human Score Agreements},
+  volume={30},
+  ISSN={1573-1839},
+  url={http://dx.doi.org/10.1007/s10956-020-09875-z},
+  DOI={10.1007/s10956-020-09875-z},
+  number={3},
+  journal={Journal of Science Education and Technology},
+  publisher={Springer Science and Business Media LLC},
+  author={Zhai,  Xiaoming and Shi,  Lehong and Nehm,  Ross H.},
+  year={2020},
+  month={nov},
+  pages={361–379}
```

Comment:
```suggestion
@article{Zhai2020meta,
  title={A Meta-Analysis of Machine Learning-Based Science Assessments: Factors Impacting Machine-Human Score Agreements},
  volume={30},
  ISSN={1573-1839},
  url={http://dx.doi.org/10.1007/s10956-020-09875-z},
  DOI={10.1007/s10956-020-09875-z},
  number={3},
  journal={Journal of Science Education and Technology},
  publisher={Springer Science and Business Media LLC},
  author={Zhai,  Xiaoming and Shi,  Lehong and Nehm,  Ross H.},
  year={2020},
  month={nov},
  pages={361–379}
}
```

This item is missing a closing brace. You might want to check for other problems in the bibliography, these might be causing the problems with citation rendering.

Reply:
Yes I think it would help. Do you know one?

From having a quick look it seems that `biber` can do what we need. `biber --tool -V references.bib`.

Reply:
nope, I would need to dig, too, or ask around.